python/tvm/relay/__init__.py

```diff
@@ -19,51 +19,56 @@
 import os
 from sys import setrecursionlimit
 
-from . import call_graph
 from . import base
 from . import ty
 from . import expr
 from . import type_functor
 from . import expr_functor
 from . import adt
-from . import analysis
+from . import prelude
+from . import loops
+from . import scope_builder
+from . import parser
+
 from . import transform
+from . import analysis
+from .analysis import call_graph, feature, alpha_equal
 from .build_module import build, create_executor, optimize
 from .transform import build_config
-from . import prelude
-from . import parser
 from . import debug
 from . import param_dict
-from . import feature
 from .backend import vm
 
 # Root operators
 from .op import Op
+from .op import nn
+from .op import image
+from .op import vision
+from .op import annotation
 from .op.reduce import *
 from .op.tensor import *
 from .op.transform import *
 from .op.algorithm import *
-from . import nn
-from . import annotation
-from . import vision
-from . import contrib
-from . import image
+from .op.nn import *
```

Comment:
Likely we do not want to expose the nn namespace to the root. Instead the relay.nn is good enough, same for other sub-namespaces under the op

Reply:
Yeah, the reason I import them was because we widely used relay.conv2d in the code. We have a nn.py before which basically doesn't the same thing as `import *` here. Same to other cases. Any other ways to achieve this?

Reply:
@tqchen Should we keep this for now?